bitnami/elasticsearch/Chart.yaml

```diff
@@ -34,4 +34,4 @@ maintainers:
 name: elasticsearch
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/elasticsearch
-version: 19.13.0
+version: 19.13.1
```

Comment:
Please revert this change after tests are 👍🏼 , there's no reason to bump chart version